src/v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -73,8 +73,18 @@
     "required": false
   },
   {
-    "destKey": "context.page",
-    "sourceKeys": ["properties.context.page", "context.page"],
+    "destKey": "context.ad.callback",
+    "sourceKeys": "properties.clickId",
+    "required": false
+  },
```

Comment:
can we also make doc update on this note of supporting only url & referrer attributes as we are restricting now ?

Reply:
Doc card is created for this.